src/test/java/nl/tudelft/ewi/devhub/server/backend/warnings/LargeCommitWarningGeneratorTest.java

```diff
@@ -114,6 +110,12 @@ public void testTooManyFiles() {
      */
     @Test
     public void testTooManyLineChanges() {
+        List<AbstractDiffModel.DiffLine> diffLines = Lists.newArrayList();
+        for (int i = 0; i < MAX_AMOUNT_OF_LINES_TOUCHED + 2; i++) {
+            AbstractDiffModel.DiffLine diffLine = new AbstractDiffModel.DiffLine(2*i, 2*i+1, String.valueOf(i));
```

Comment:
As you set both the `newLineNumber` and the `oldLineNumber` these lines are assumed to be *unchanged* (as a modified line would be split up in to the removal of the old version, and the addition of the new version of the line). As they are unchanged, they are not counted for the warning (by design). You therefore have to adjust the test so that these lines are either *added* to or *removed* from the file. You can do this by setting one of the counts to `null`.